### ISSUE TITLE
Template init error due to bad influxdb select.

### ISSUE THREAD

**Issue by JeffKitson** [lephisto/tesla-apiscraper#18]
I experienced this error when starting up the dashboards:

Templating init failed {message: "InfluxDB Error: error parsing query: found (, expected identifier at line 1, char 36", data: {…}, config: {…}}

I tried running the select query in the influx cli and the query still failed. I'm not sure if it's a result of my instillation since I'm not very familiar with influx but I found a workaround of setting display_name to a constant inside grafana. When this query is bypassed for every dashboard the rest of the renderings work as expected.

**Comment by zax123** [comment]
@JeffKitson could you please elaborate where exactly you put that display_name setting?  I have the same problem.  Thanks!

**Comment by JeffKitson** [comment]
@zax123 Sorry vehicle_name is actually the variable that needs to be edited. in the settings menu for each individual dashboard you'll find :
![vehicle_name](https://user-images.githubusercontent.com/4663739/51610518-f0a47100-1eea-11e9-8b0b-fe9a272d69d4.PNG)

**Comment by zax123** [comment]
Thank you sir! Now I just need to get my DB authentication working again. :P  I upgraded to 1.7.x and it broke authentication. :)  What is this weird stuff?  influx and grafana... :)

**Comment by lephisto** [comment]
You can wether set it static, otherwise it will be filled from the DB dynamically once you have Data in it (in case of multiple vehicles in one DB)

**Comment by JeffKitson** [comment]
I had about 12-24 hours of data and was still experiencing this error. It's not a big error with the work around. I haven't investigated the issue more today but I'm wondering if it's an issue relating to multiple cars vs a single car. I have one Tesla and was able to select the target db field for the vehicle name in cli queries.

**Comment by JeffKitson** [comment]
Yes, that's the name I substitute as a constant "Darrell". It's been the vehicles name since I got it and I can see it in select * queries so I'm not sure why this query is failing. I've notices i can't seem to select display_name from any table which seems odd.
